src/main/resources/config.yml

```diff
@@ -26,4 +26,5 @@ hurtBlocks:
   - BLACK_BED
   - SCAFFOLDING
 applyFallForce: false
-damageMultiplier : 1
\ No newline at end of file
+damageMultiplier : 1
+wetDamageMultipler : 1
```

Comment:
Could we name this `waterfallDamageMultiplier` since this doesn't affect damage when falling into water, only swimming up waterfalls?